lib/ansible/executor/module_common.py

```diff
@@ -1319,7 +1319,23 @@ def modify_module(module_name, module_path, module_args, templar, task_vars=None
     return (b_module_data, module_style, shebang)
 
 
-def get_action_args_with_defaults(action, args, defaults, templar):
+def get_action_args_with_defaults(action, args, defaults, templar, redirected_names=None):
+    collection_groups = {
+        'acme': ['community.crypto'],
+        'aws': ['amazon.aws', 'community.aws'],
+        'azure': ['azure.azcollection'],
+        'cpm': ['wti.remote'],
+        'docker': ['community.general'],
+        'gcp': ['google.cloud'],
+        'k8s': ['community.kubernetes', 'community.general'],
+        'os': ['openstack.cloud'],
+        'ovirt': ['ovirt.ovirt', 'community.general'],
+        'vmware': ['community.vmware'],
+        'testgroup': ['testns.testcoll']
```

Comment:
this will repro the "what if one of the collections is missing?" case

```suggestion
        'testgroup': ['testns.testcoll', 'testns.othercoll', 'testns.boguscoll']
```